src/components/editor/EditAuthorsForm.jsx

```diff
@@ -84,8 +41,8 @@ export default class EditAuthorsForm extends BaseComponent {
               <div key={author.slug}>
                 {
                   !striked
-                    ? <a style={{float: "left", cursor: "pointer"}} aria-label="Remove Author Button" onClick={this.handleDeleteAuthor.bind(this, author.slug, true)}>&times;&nbsp;</a>
-                    : <a style={{float: "left", cursor: "pointer"}} aria-label="Remove Author Button" onClick={this.handleUnstrikeAuthor.bind(this, author.slug)}>~&nbsp;</a>
+                    ? <a style={{float: "left", cursor: "pointer"}} aria-label="Remove Author Button" onClick={this.props.handleDeleteAuthor.bind(null, author.slug, true)}>&times;&nbsp;</a>
```

Comment:
And about the href=# again.

1. I don't like that it sends it to `currentPath#'
2. If I use href=# I need to make more styles anyway as I need to set colour and textDecorator

So I think I'll keep it this way if that's okay with you.